kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/SyncSchedule/EditSyncSchedule.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+  >
+    <KPageContainer>
+      <KGrid gutter="48">
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+        <KGridItem>
+          <p>{{ $tr('deviceName') }}</p><br>
+        </KGridItem>
+
+        <KGridItem>
+          <KDropdownMenu
+            text="Every After 5 minutes"
+            appearance="raised-button"
+            position="relative"
+          />
+        </KGridItem>
+        <br>
+
+        <KGridItem>
+          <p>{{ $tr('serverTime') }} </p>
+        </KGridItem>
+
+        <KGridItem>
+          <KCheckbox>
+            {{ $tr('checboxlabel') }}
+          </KCheckbox>
+        </KGridItem>
+        <KGridItem>
+          <KButton
+            appearance="basic-link"
+            @click="removeDevice"
+          >
+            {{ msg }}
+          </KButton>
+        </KGridItem>
+
+      </KGrid>
+    </KPageContainer>
+    <KPageContainer />
+
+    <KModal
+      v-if="removeDeviceModal"
+      :title="$tr('removeDevice')"
+      size="medium"
+      submitText="REMOVE"
+      cancelText="CANCEL"
+      @cancel="closeModal"
+      @submit="ConfirmRemoveDevice"
+    >
+      <KGrid>
+        <KGridItem
+          :layout8="{ span: 4 }"
+          :layout12="{ span: 6 }"
+        >
+          <p>{{ $tr('deviceName') }}</p>
+        </KGridItem>
+
+        <KGridItem
+          :layout8="{ span: 8 }"
+          :layout12="{ span: 12 }"
+        >
+          <p>{{ $tr('removeDeviceWarning') }}</p>
+          <p>{{ $tr('deviceNotConnected') }}</p>
+        </KGridItem>
+
+      </KGrid>
+    </KModal>
+  </ImmersivePage>
+
+</template>
+
+
+<script>
+
+  import ImmersivePage from 'kolibri.coreVue.components.ImmersivePage';
+
+  export default {
+    name: 'EditSyncSchedule',
+    components: {
+      ImmersivePage,
+    },
+    data() {
+      return { removeDeviceModal: false };
+    },
+    methods: {
+      removeDevice() {
+        this.removeDeviceModal = true;
+      },
+      closeModal() {
+        this.removeDeviceModal = false;
+      },
+      ConfirmRemoveDevice() {
+        this.removeDeviceModal = false;
```

Comment:
This is a good start to have a function here for confirm! The next step is to actually save data when submitting. To get you started for context, here is an example you can look at. [On submit, create lesson is called.](https://github.com/MisRob/kolibri/blob/9ffeca2a815719bda629affe011fa67e37531ee3/kolibri/plugins/coach/assets/src/views/plan/LessonCreationPage/index.vue#L27). Then, [this method is responsible for actually saving the data](https://github.com/MisRob/kolibri/blob/9ffeca2a815719bda629affe011fa67e37531ee3/kolibri/plugins/coach/assets/src/views/plan/LessonCreationPage/index.vue#L57). We will talk more about how this works, but reviewing this is a good place to start. See if you can find the different functions that are included here - `createLesson`, `showSnackbarNotification`, and the error handling.